[----------] Running tests from PanickingTest
[ RUN      ] PanickingTest.PanickingTest
panicking_test.go:39:
panic: foobar

github.com/jacobsa/ogletest/somepkg_test.(*PanickingTest).PanickingTest
	some_file.txt:0
reflect.Value.call
	some_file.txt:0
reflect.Value.Call
	some_file.txt:0
github.com/jacobsa/ogletest.runMethodIfExists
	some_file.txt:0
github.com/jacobsa/ogletest.runTest
	some_file.txt:0
github.com/jacobsa/ogletest.runTestsInternal
	some_file.txt:0
github.com/jacobsa/ogletest._func_002
	some_file.txt:0
sync.(*Once).Do
	some_file.txt:0
github.com/jacobsa/ogletest.RunTests
	some_file.txt:0
github.com/jacobsa/ogletest/somepkg_test.TestPanickingTest
	some_file.txt:0
testing.tRunner
	some_file.txt:0
runtime.goexit
	some_file.txt:0


[  FAILED  ] PanickingTest.PanickingTest
[ RUN      ] PanickingTest.ZzzSomeOtherTest
[       OK ] PanickingTest.ZzzSomeOtherTest
[----------] Finished with tests from PanickingTest
--- FAIL: somepkg (1.23 seconds)
FAIL
exit status 1
FAIL somepkg 1.234s
